pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pytestCheckHook,
+  setuptools,
+  cachecontrol,
+  lxml-html-clean,
+  requests,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rajatomar788";
+    repo = "pywebcopy";
+    rev = "v${version}";
+    hash = "sha256-XTPk3doF9dqImsLtTB03YKMWLzQrJpJtjNXe+691rZo=";
+  };
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    cachecontrol
+    lxml-html-clean
+    requests
+    six
+  ];
+
```

Comment:
Adding `pythonImportsCheck` should be a good smoke test.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck

Reply:
@natsukium can you please create a commit suggestion?

Reply:
something like 
```
index 31aeedcd7807..e8d8cb61a334 100644
--- a/pkgs/development/python-modules/pywebcopy/default.nix
+++ b/pkgs/development/python-modules/pywebcopy/default.nix
@@ -24,6 +24,8 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [ pytestCheckHook ];
 
+  pythonImportsCheck = [ "pywebcopy" ];
+
   build-system = [ setuptools ];
 
   dependencies = [

```

Reply:
Fixed